### PR TITLE
Create index.md

### LINKED ISSUE #6799: [CSS] Designcember: Interop 2022 [12/17/2021]
* Proposed title: Designcember: Interop 2022
* Proposed by: foolip
* Related areas: CSS
* Content type: None of the above
* Deadline: 12/17/2021
* Deadline explanation: Per Designcember schedule
* Content summary: What Chrome and other browser vendors are doing to improve browser compatibility on selected features in 2022

### PR DESCRIPTION
**Pull request by robnyman**
Fixes #6799

Changes proposed in this pull request:

- Blog post draft
-
-

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

File: src/site/content/en/blog/interop-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+subhead:
+    "For the first time ever, all major browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2022 will improve the experience of developing for the web in 15 key areas. In this article, find out how we got here, what the project focuses on, how success will be measured, and how you can track progress."
+description:
+    "Learn more about how all browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2022 will improve the experience of developing for the web in 15 key areas."
+authors:
+  - rachelandrew
+  - robertnyman
+  - foolip
+hero: "image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/xv75zUKa4jUR2xGMZ7Xb.jpg"
+alt: "A picture of a brick wall"
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:13]:
```suggestion
alt: "A picture of a brick wall."
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+date: 2022-02-16
+updated: 2022-02-16
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:15]:
```suggestion
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    "For the first time ever, all major browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2022 will improve the experience of developing for the web in 15 key areas. In this article, find out how we got here, what the project focuses on, how success will be measured, and how you can track progress."
+description:
+    "Learn more about how all browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2022 will improve the experience of developing for the web in 15 key areas."
+authors:
+  - rachelandrew
+  - robertnyman
+  - foolip
+hero: "image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/xv75zUKa4jUR2xGMZ7Xb.jpg"
+alt: "A picture of a brick wall"
+date: 2022-02-16
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:14]:
```suggestion
date: 2022-02-22
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  - blog
+  - css
+---
+
+For the first time ever, all major browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2022 will improve the experience of developing for the web in 15 key areas. In this article, find out how we got here, what the project focuses on, how success will be measured, and how you can track progress.
+
+
+## It all started in 2019
+
+Back in 2019 Mozilla, Google and others started [a major effort](https://insights.developer.mozilla.org/) to understand developers’ pain points, in the form of the [MDN Developer Needs Assessment surveys](https://insights.developer.mozilla.org/), and the deep-dive [Browser Compatibility Report](https://insights.developer.mozilla.org/reports/mdn-browser-compatibility-report-2020.html). These reports gave us detailed and actionable information to address top challenges for developers with the web platform, and led to the [Compat 2021 effort](/compat2021/).
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:26]:
```suggestion
Back in 2019 Mozilla, Google, and others started [a major effort](https://insights.developer.mozilla.org/) to understand developers' pain points, in the form of the [MDN Developer Needs Assessment surveys](https://insights.developer.mozilla.org/), and the deep-dive [Browser Compatibility Report](https://insights.developer.mozilla.org/reports/mdn-browser-compatibility-report-2020.html). These reports gave us detailed and actionable information to address top challenges for developers with the web platform, and led to the [Compat 2021 effort](/compat2021/).
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+We were delighted to reach [a score of over 90%](/compat2021-holiday-update/) across all implementations at the end of 2021!
+
+## What is Interop 2022?
+
+Interop 2022 is a benchmark, agreed on by representatives of three major browser implementations, and developed through a process of [public nomination](https://github.com/web-platform-tests/interop-2022/labels/proposal) and review with input from [supporters](https://github.com/web-platform-tests/interop-2022/blob/main/supporters.md) [Apple](https://webkit.org/blog/12288/working-together-on-interop-2022/), [Bocoup](https://bocoup.com/blog/interop-2022), Google, [Igalia](https://igalia.com/2022/02/22/Igalia-and-Interop-2022), [Microsoft](https://aka.ms/microsoft-interop2022) and [Mozilla](https://hacks.mozilla.org/2022/02/interop-2022/).
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:34]:
```suggestion
Interop 2022 is a benchmark, agreed on by representatives of three major browser implementations, and developed through a process of [public nomination](https://github.com/web-platform-tests/interop-2022/labels/proposal) and review with input from [supporters](https://github.com/web-platform-tests/interop-2022/blob/main/supporters.md) [Apple](https://webkit.org/blog/12288/working-together-on-interop-2022/), [Bocoup](https://bocoup.com/blog/interop-2022), Google, [Igalia](https://igalia.com/2022/02/22/Igalia-and-Interop-2022), [Microsoft](https://aka.ms/microsoft-interop2022), and [Mozilla](https://hacks.mozilla.org/2022/02/interop-2022/).
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+- `color-contrast()`: Selects from a list of colors the color with the highest contrast to a specified single color.
+
+These functions support expanded color spaces (LAB, LCH, and P3), in addition to HSL and sRGB, they default to the uniform LCH color space. 
+
+### New viewport units
+
+Difficulties dealing with viewport sizing are prominent in both the [MDN Browser Compatibility Report 2020](https://insights.developer.mozilla.org/reports/mdn-browser-compatibility-report-2020.html#findings-viewport) and the new [State of CSS 2021](https://2021.stateofcss.com/en-US/opinions/#browser_interoperability_features) survey. [CSS Values and Units Level 4](https://drafts.csswg.org/css-values-4/#viewport-relative-lengths) adds new units for the largest, smallest, and dynamic viewport sizes, `lv*`, `sv*`, and `dv*`. These units will make it easier to create layouts that fill the visible viewport on mobile devices while taking the address bar into account.
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/JUvvIgXen1zmHFH53CBS.png", alt="AN image showing the different parts of the viewport for each type of viewport unit", width="800", height="664" %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:65]:
```suggestion
{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/JUvvIgXen1zmHFH53CBS.png", alt="The different parts of the viewport for each type of viewport unit.", width="800", height="664" %}
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+
+Additionally, the cross vendor team behind Interop 2022 will collaborate on researching and improving the state of interoperability of existing viewport measurement features, including the existing `vh` unit.
+
+### Scrolling
+
+The [2021 Scroll Survey Report](/2021-scroll-survey-report/) confirms that scrolling features and scrolling compatibility are difficult to implement and have many areas for improvement. We’ll focus on [scroll snap](https://developer.mozilla.org/docs/Web/CSS/CSS_Scroll_Snap), [scroll-behavior](https://developer.mozilla.org/docs/Web/CSS/scroll-behavior), and [overscroll-behavior](https://developer.mozilla.org/docs/Web/CSS/overscroll-behavior) to help scrolling be more consistent and smooth across platforms.
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:71]:
```suggestion
The [2021 Scroll Survey Report](/2021-scroll-survey-report/) confirms that scrolling features and scrolling compatibility are difficult to implement and have many areas for improvement. We'll focus on [scroll snap](https://developer.mozilla.org/docs/Web/CSS/CSS_Scroll_Snap), [scroll-behavior](https://developer.mozilla.org/docs/Web/CSS/scroll-behavior), and [overscroll-behavior](https://developer.mozilla.org/docs/Web/CSS/overscroll-behavior) to help scrolling be more consistent and smooth across platforms.
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+We are also exploring new [scroll snap feature proposals](https://github.com/argyleink/ScrollSnapExplainers). 
+
+
+### Subgrid
+
+The [`subgrid`](https://developer.mozilla.org/docs/Web/CSS/CSS_Grid_Layout/Subgrid) value of `grid-template-columns` and `grid-template-rows` means that a grid item that has `display: grid` applied can inherit the track definition from the part of the parent grid it is placed over.
+
+For example, the following three card components have a header and footer aligned with the adjacent card headers and footers, even though each card has an independent grid. This pattern works because each card is an item that spans three rows of the parent grid, then uses subgrid to inherit those rows into the card.
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/QIrr5x5M7qCIo8rHMpOC.png", alt="A picture of a three card component where the headers and footers are aligned between the cards", width="800", height="205" %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:82]:
```suggestion
<figure>
{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/QIrr5x5M7qCIo8rHMpOC.png", alt="A three card component where the headers and footers are aligned between the cards.", width="800", height="205" %}
  <figcaption><a href="https://codepen.io/web-dot-dev/pen/XWzqrLR">See this on CodePen</a>.</figcaption>
</figure>
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+- Grid
+- Sticky Positioning
+- Transforms
+
+
+### Investigation efforts
+
+In addition to the 15 focus areas, Interop 2022 includes three investigation efforts. These are areas that are problematic and need improvement, but where the current state of the specifications or tests aren’t yet good enough to be able to score progress using test results:
+
+- Editing, contenteditable and execCommand
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:107]:
```suggestion
- Editing, `contenteditable` and `execCommand`
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+- Pointer and Mouse Events
+- Viewport Measurement
+
+Browser vendors and other stakeholders will collaborate on improving the tests and specifications for these areas, so that they could be included in future iterations of this effort.
+
+
+## Measuring success and tracking progress
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/YSLThucidlfsbtruHrkK.png", alt="An image showing scores per browser", width="800", height="366" %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:116]:
```suggestion
{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/YSLThucidlfsbtruHrkK.png", alt="The scores per browser - 72 for Chrome and Edge, 74 for Firefox, 72 for Safari Technology Preview.", width="800", height="366" %}
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+
+The existing [web-platform-tests dashboard](https://wpt.fyi/) will be used to track the progress in the 15 focus areas. For each area, a set of tests have been identified. Browsers are then scored against these tests, giving a score for each area and an overall score for all 15 areas.
+
+To follow along and track the progress, check out the [Interop 2022 dashboard](https://wpt.fyi/interop-2022). Over the course of the year, you can follow along and see how the platform you build for is improving. 
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/FWFA2iKpQqJc3KpEHIij.png", alt="An image of a table with scores for many areas for all major web browsers", width="800", height="530" %}
+
+## What will all this mean for developers?
+
+The goal of these multi-year interoperability efforts, in the form of Compat 2021, Interop 2022 and much more, is to fully acknowledge and address the pain points developers have experienced through for many years. And it’s not a one-browser effort but rather a strong collaboration between all major browser vendors and friends for improving the web platform across the board.
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:126]:
```suggestion
The goal of these multi-year interoperability efforts, in the form of Compat 2021, Interop 2022 and much more, is to fully acknowledge and address the pain points developers have experienced through for many years. And it's not a one-browser effort but rather a strong collaboration between all major browser vendors and friends for improving the web platform across the board.
```

File: src/site/content/en/blog/interop-2022/index.md [hunk 12]
```diff
@@ -0,0 +1,142 @@
...
+
+## Measuring success and tracking progress
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/YSLThucidlfsbtruHrkK.png", alt="An image showing scores per browser", width="800", height="366" %}
+
+The existing [web-platform-tests dashboard](https://wpt.fyi/) will be used to track the progress in the 15 focus areas. For each area, a set of tests have been identified. Browsers are then scored against these tests, giving a score for each area and an overall score for all 15 areas.
+
+To follow along and track the progress, check out the [Interop 2022 dashboard](https://wpt.fyi/interop-2022). Over the course of the year, you can follow along and see how the platform you build for is improving. 
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/FWFA2iKpQqJc3KpEHIij.png", alt="An image of a table with scores for many areas for all major web browsers", width="800", height="530" %}
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:122]: What is this table? It doesn't say what it is, where it has come from. If there is a link to the actual table then add it as a caption. Or, if the data is important use an actual table. Right now it doesn't have any context.

File: src/site/content/en/blog/interop-2022/index.md [hunk 13]
```diff
@@ -0,0 +1,142 @@
...
+
+### Subgrid
+
+The [`subgrid`](https://developer.mozilla.org/docs/Web/CSS/CSS_Grid_Layout/Subgrid) value of `grid-template-columns` and `grid-template-rows` means that a grid item that has `display: grid` applied can inherit the track definition from the part of the parent grid it is placed over.
+
+For example, the following three card components have a header and footer aligned with the adjacent card headers and footers, even though each card has an independent grid. This pattern works because each card is an item that spans three rows of the parent grid, then uses subgrid to inherit those rows into the card.
+
+{% Img src="image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/QIrr5x5M7qCIo8rHMpOC.png", alt="A picture of a three card component where the headers and footers are aligned between the cards", width="800", height="205" %}
+
+Codepen: [https://codepen.io/rachelandrew/pen/yLzgowp](https://codepen.io/rachelandrew/pen/yLzgowp)
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:84]:
```suggestion
```

File: src/site/content/en/blog/interop-2022/index.md (hunk 12 above)
Comment (robnyman) [reply src/site/content/en/blog/interop-2022/index.md]: Added a caption. Data is not the important, but the visual is, to get a feel for what the dashboard looks like.

**Review by robnyman — COMMENTED** [review]
Changes done

**Review by robnyman — COMMENTED** [review]
Looks good!

**Review by rachelandrew — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/interop-2022/index.md [hunk 14]
```diff
@@ -0,0 +1,146 @@
...
+  - foolip
+hero: "image/RYmV5NPuMZRoF3PVwIXTUpdYeQ23/xv75zUKa4jUR2xGMZ7Xb.jpg"
+alt: "A picture of a brick wall."
+date: 2022-03-03
+tags:
+  - blog
+  - css
+---
+
+For the first time ever, all major browser vendors, and other stakeholders, have come together to solve the top browsers compatibility issues identified by web developers. Interop 2022 will improve the experience of developing for the web in 15 key areas. In this article, find out how we got here, what the project focuses on, how success will be measured, and how you can track progress.
```
Comment (rachelandrew) [inline src/site/content/en/blog/interop-2022/index.md:20]: We probably want to remove this because it's repetitive of the subhead (or change the subhead).

Comment (robnyman) [reply src/site/content/en/blog/interop-2022/index.md:20]: Absolutely. I didn't know if the subhead will be visible in the post or not, hence the repetition.

**Pull request merged**